test/unit/org/apache/cassandra/cql3/validation/operations/AlterTest.java

```diff
@@ -241,7 +304,7 @@ public void testAlterKeyspaceWithNoOptionThrowsConfigurationException() throws T
 
         // Try to alter the created keyspace without any option
         assertInvalidThrow(ConfigurationException.class, "ALTER KEYSPACE testABC WITH replication={ 'class' : 'NetworkTopologyStrategy' }");
-        assertInvalidThrow(ConfigurationException.class, "ALTER KEYSPACE testXYZ WITH replication={ 'class' : 'SimpleStrategy' }");
+        schemaChange("ALTER KEYSPACE testXYZ WITH replication={ 'class' : 'SimpleStrategy' }");
```

Comment:
Is this expected change?

Reply:
yes, since, with this change, ignoring replication_factor for simple strategy is valid, and does not throw an exception

Reply:
👍

Reply:
May want to update the comment to indicate why this is ok not to fail?